lib/src/executable.dart

```diff
@@ -161,7 +161,7 @@ transformers:
     if (signalSubscription == null) return;
     signalSubscription.cancel();
     signalSubscription = null;
-    stdinLines.cancel(immediate: true);
+    await stdinLines.cancel(immediate: true);
```

Comment:
I don't think this should matter, and it would be nice to get the runner started closing as soon as possible. Also, since this is in the signal-handling infrastructure, it's definitely not causing bot issues.